scratch/sample_metadata.py

```diff
@@ -0,0 +1,857 @@
+[if timeseries] Time series of electricity demand by planning area.
+# [if assn] Links {source1} to {source2}.
+# [if codes] Explains codes for {purpose}.
+# [if entity/scd] Entity/SCD table for {short_description}.
+# [else] {Concise verb/noun indicating type} {short_description}.
+
+Derived from FERC Form 714, Part III, Schedule 2a.
+
+Usage Warnings
+^^^^^^^^^^^^^^
+This table includes data from the pre-2021 CSV raw source as well as the newer 2021 through present XBRL raw source.
+
+There was some cleaning done to the datetime_utc timestamps. The Form 714 includes sparse documentation for respondents for how to interpret timestamps. The form asks respondents to provide 24 instances of hourly demand for each day. The form is labeled with hour 1-24. There is no indication if hour 1 begins at midnight.
+
+The XBRL data contained several formats of timestamps. Most records corresponding to hour 1 of the Form have a timestamp with hour 1 as T1. About two thirds of the records in the hour 24 location of the form have a timestamp with an hour reported as T24 while the remaining third report this as T00 of the next day. T24 is not a valid format for the hour of a datetime, so we convert these T24 hours into T00 of the next day. A smaller subset of the respondents reports the 24th hour as the last second of the day - we also convert these records to the T00 of the next day.
+
+Additional Details
+^^^^^^^^^^^^^^^^^^
+This table includes three respondent ID columns: one from the CSV raw source, one from the XBRL raw source and another that is PUDL-derived that links those two source ID's together. This table has filled in source IDs for all records so you can select the full timeseries for a given respondent from any of these three IDs.
+""",
+        "schema": {
+            "fields": [
+                "respondent_id_ferc714",
+                "respondent_id_ferc714_csv",
+                "respondent_id_ferc714_xbrl",
+                "report_date",
+                "datetime_utc",
+                "timezone",
+                "demand_mwh",
+            ],
+            "primary_key": ["respondent_id_ferc714", "datetime_utc"],
+        },
+        "sources": ["ferc714"],
+        "field_namespace": "ferc714",
+        "etl_group": "ferc714",
+        "create_database_schema": False,
+    },
+    "out_ferc714__respondents_with_fips": {
+        "original_description": """
+Annual respondents with the county FIPS IDs for their service territories.
+""",
+        "description": """
+[if timeseries] Time series of {short_description}.
+[if assn] Links {source1} to {source2}.
+[if codes] Explains codes for {purpose}.
+[if entity/scd] Entity/SCD table for FERC 714 respondents, annotated with the county FIPS IDs for their service territories.
+[else] {Concise verb/noun indicating type} {short_description}.
+
+This table has no primary key. Each row represents a respondent's information for a particular report date.
+
+Derived from ferc714.
```

Comment:
This is a good example of a table that I'm like what the heck is this for. Why do we need this table that's randomly annotated with the county FIPS ID? Did we make it for something specific?